apis/projectcontour/v1alpha1/contourconfig_helpers.go

```diff
@@ -48,6 +52,51 @@ func (c *ContourConfigurationSpec) Validate() error {
 	return nil
 }
 
+func (t *TracingConfig) Validate() error {
+	if t.OverallSampling == nil && t.MaxPathTagLength == nil && t.ExtensionService == nil && t.CustomTags == nil {
+		return nil
+	}
```

Comment:
Is there any specific reason to return nil for this case? I think that if TracingConfig is non-nil, then it should require at least an ExtensionService to be specified. I'd just drop these lines, and adjust the associated unit test case